subsys/bluetooth/host/att.c

```diff
@@ -1892,6 +2207,10 @@ static const struct att_handler {
 		sizeof(struct bt_att_indicate),
 		ATT_INDICATION,
 		att_indicate },
+	{ BT_ATT_OP_NOTIFY_MULT,
+		sizeof(struct bt_att_notify_mult),
+		ATT_NOTIFICATION,
+		att_notify_mult },
```

Comment:
This is not an EATT operation? (no #ifdef around it)

Reply:
No, the new procedures are not exclusive to EATT bearer, they just need to be enable by the client but can be used on ATT bearer as well.